spec/factories/notes.rb

```diff
@@ -53,7 +53,7 @@
     end
 
     trait :with_attachment do
-      attachment { fixture_file_upload(Rails.root + "spec/fixtures/dk.png", "`/png") }
+      attachment { Rack::Test::UploadedFile.new(Rails.root + "spec/fixtures/dk.png", "`/png") }
```

Comment:
Line is too long. [95/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.